frontend/test/unit/specs/components/AudioTrack/v-audio-track.spec.js

```diff
@@ -19,9 +19,6 @@ jest.mock("~/composables/use-match-routes", () => ({
   useMatchSingleResultRoutes: jest.fn(() => ({ matches: { value: false } })),
 }))
 
-jest.mock("~/composables/use-browser-detection", () => ({
-  useBrowserIsBlink: jest.fn(() => false),
-}))
 
```

Comment:
Please also remove this blank line. That should make the lint check pass.

Reply:
Ok Done.